app-backend/db/src/main/scala/SceneToLayerDao.scala

```diff
@@ -0,0 +1,239 @@
+package com.rasterfoundry.database
+
+import java.util.UUID
+
+import cats.Applicative
+import cats.data.{NonEmptyList => NEL, _}
+import cats.implicits._
+import com.rasterfoundry.database.Implicits._
+import com.rasterfoundry.database.util.RFTransactor
+import com.rasterfoundry.datamodel.{
+  BatchParams,
+  ColorCorrect => RFColorCorrect,
+  MosaicDefinition,
+  SceneToLayer,
+  SceneToLayerWithSceneType
+}
+import com.typesafe.scalalogging.LazyLogging
+import com.rasterfoundry.datamodel._
+import doobie._
+import doobie.Fragments._
+import doobie.implicits._
+import doobie.postgres._
+import doobie.postgres.implicits._
+import fs2.Stream
+import geotrellis.vector.{MultiPolygon, Polygon, Projected}
+import io.circe.syntax._
+
+case class SceneToLayerDao()
+
+object SceneToLayerDao extends Dao[SceneToLayer] with LazyLogging {
+
+  val tableName = "scenes_to_layers"
+
+  val selectF: Fragment = sql"""
+    SELECT
+      scene_id, project_layer_id, accepted, scene_order, mosaic_definition
+    FROM
+  """ ++ tableF
+
+  def acceptScene(projectLayerId: UUID, sceneId: UUID): ConnectionIO[Int] = {
+    fr"""
+      UPDATE scenes_to_layers
+      SET accepted = true
+      WHERE project_layer_id = ${projectLayerId}
+        AND scene_id = ${sceneId}
+    """.update.run
+  }
+
+  def acceptScenes(projectLayerId: UUID,
+                   sceneIds: List[UUID]): ConnectionIO[Int] = {
+    sceneIds.toNel match {
+      case Some(ids) => acceptScenes(projectLayerId, ids)
+      case _         => 0.pure[ConnectionIO]
+    }
+  }
+
+  def acceptScenes(projectLayerId: UUID,
+                   sceneIds: NEL[UUID]): ConnectionIO[Int] = {
+    (
+      fr"""
+        UPDATE scenes_to_layers
+        SET accepted = true
+      """ ++ Fragments.whereAnd(
+        fr"project_layer_id = ${projectLayerId}",
+        Fragments.in(fr"scene_id", sceneIds)
+      )
+    ).update.run
+  }
+
+  def addSceneOrdering(projectId: UUID): ConnectionIO[Int] = {
+    (fr"""
+      UPDATE scenes_to_layers
+      SET scene_order = s.rnum
+      FROM (
+        SELECT scene_id, project_layer_id, row_number() over (ORDER BY stl.scene_order ASC, scenes.acquisition_date ASC, scenes.cloud_cover ASC) as rnum
+        FROM scenes_to_layers stl
+          JOIN scenes ON scenes.id = stl.scene_id
+          JOIN project_layers pl ON pl.id = stl.project_layer_id
+          WHERE pl.project_id = ${projectId}
+      ) s
+      WHERE scenes_to_layers.project_layer_id = s.project_layer_id
+        AND scenes_to_layers.scene_id = s.scene_id
+    """).update.run
+  }
+
+  def moveSceneOrder(projectLayerId: UUID,
```

Comment:
If the route is commented out and we don't need it can you remove the route, this function, and update the spec